sampleapi/api_ok.go

```diff
@@ -1,10 +1,32 @@
 package sampleapi
 
+import (
+	"fmt"
+	"sync"
+)
+
 // HelloWorld ...
 func HelloWorld() {
+
+	wg := sync.WaitGroup()
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		fmt.Printf("test goroutine")
+	}()
+	wg.Wait()
+
 	for i := 0; i < 10; i++ {
 		for j := 0; j < 10; j++ {
 			fmt.Printf("Hello, world!")
 		}
 	}
+
```

Comment:
The code looks correct and is very straightforward. A few suggestions to improve the code readability are:

1. Use descriptive variable names - Instead of using "i" as the variable, use something more descriptive like "iteration".
2. Use more meaningful output - Instead of just printing "Test Output" and "Test Nihao" each time, use a variable to store the output that is more meaningful.
3. Remove redundant code - The two for loops are almost identical. Combine them into one to reduce redundancy.